pkg/model/bundle.go

```diff
@@ -211,14 +216,21 @@ func GetArchivePathComponents(archivePath string) (ArchivePathComponents, error)
 	//return ArchivePathComponents{}, fmt.Errorf("path is invalid: %v, path: %s", cs, archivePath)
 }
 
+// GetBundleTimeStamp yields the current UTC time
 func GetBundleTimeStamp() time.Time {
 	t := time.Now()
 	return t.UTC()
 }
 
+// IsGeneratedFile indicate if some file comes from auto-generation (e.g. .datamon files)
 func IsGeneratedFile(file string) bool {
 	// TODO: Need to find a way for AeroFs to convert to abs patch while honoring the fake root
 	//path, err := filepath.Abs(file)
-	match, _ := regexp.MatchString("^.datamon/*|^/.datamon/*|^/.datamon$|^.datamon$|^./.datamon/*|^./.datamon$", file)
-	return match
+	return genFileRe.MatchString(file)
+}
+
+func init() {
+	metaRe = regexp.MustCompile(`^\.datamon/(.*)\.yaml$`)
```

Comment:
this is a good catch for performance and catching errors early on.  i'd feel slightly better if all the regular expressions were attached to a structure so that the global names aren't too numerous.. yet another nitpick to be resolved time permitting.